### ISSUE TITLE
ROBOT report - need to check that we have no clashes with HPO

### ISSUE THREAD

**Issue by nicolevasilevsky** [monarch-initiative/mondo#1499]
Per our [Mondo call on 05/08](https://docs.google.com/document/d/1wXBv1Bh4JmdqPCbQyu78ZyrITpxQTAQi1odNO-0aPcw/edit), could we get a check in the ROBOT report to check that Mondo terms do not have any clashes with HPO terms.

For example:
MONDO_0001627 'dementia (disease)'
MONDO_0002280 'anemia (disease)'

**Comment by nicolevasilevsky** [comment]
The issue here is that we have terms in Mondo that are diseases and sometimes there are terms in Human Phenotype Ontology (HPO) that have the same label. Examples:
MONDO_0001627 dementia and HP:0000726 Dementia
MONDO_0002280 anemia and HP:0001903 Anemia

You can view the entire HPO here: https://hpo.jax.org/app/ (or download the owl file [here](https://github.com/obophenotype/human-phenotype-ontology/releases/tag/v2021-10-10)

I can't remember what our ultimate goal is though - do we not want to have overlapping terms in Mondo and HPO?

I'll add this to the QC call agenda.

**Comment by psiotwo** [comment]
Not sure whether it is related to this issue, but e.g. the following MONDO concepts share a definition with HPO concepts: MONDO:0005260, MONDO:0001014.

**Comment by matentzn** [comment]
It fits in here! But its a slightly adjacent issue. Thank your for reporting!

**Comment by sabrinatoro** [comment]
This issue was created more than 2 years ago and has not been addressed to date. It has therefore been identified as a candidate to be closed without action.
If anyone believes that this issue should be addressed, please comment here.
Otherwise, this issue will automatically be closed with the October 2025 release.

@matentzn is this something that should be addressed, or can we close this issue?

**Comment by matentzn** [comment]
We can close this as being an issue whose resolution is too complicated to be worth it. We have tried it multiple times, through dozen of curator hours at it. The truth is, the boundary between disease and phenotype my be conceptually sound, but practically so hazy and hard to recognise that it would take huge amounts of curator hours to handle this, for few tangible benefits. I also vote to close this.
